### ISSUE TITLE
ZIP unpack/repack implementation

### ISSUE THREAD

**Issue by Grossley** [UnderminersTeam/UndertaleModTool#442]
* For scripts
* For profile system

**Comment by Miepee** [comment]
Are there any objections in closing this issue as wontfix? I'm not particurlaly interested in it.

**Comment by colinator27** [comment]
Yeah, this is probably something that we don't need to reimplement as APIs ourselves right now.
